### PR TITLE
add permissions docs

### PR DESCRIPTION
**Pull request by solomon-negusse**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
- [ ] Make sure you are making a pull request against the develop branch (left side). Also you should start your branch off our develop.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: app/routes/datasets/dataset.py [hunk 1]
```diff
@@ -55,7 +55,11 @@ async def create_dataset(
     is_authorized: bool = Depends(is_admin),
     response: Response,
 ) -> DatasetResponse:
-    """Create or update a dataset."""
+    """Create a dataset. a “dataset” is largely a metadata concept: it represents
```
Comment (danscales) [inline app/routes/datasets/dataset.py:58]: Change 'a' -> 'A'.

File: app/routes/datasets/dataset.py [hunk 2]
```diff
@@ -55,7 +55,11 @@ async def create_dataset(
...
+    a data product that may have multiple versions or file formats over time.
+
+    This operation requires a `MANAGER` or an `ADMIN` user role.
```
Comment (danscales) [inline app/routes/datasets/dataset.py:61]: Maybe add an extra sentence here:

The user that creates a dataset using this operation becomes the `owner` of the dataset, which provides the user with the privileges to do further write operations on the dataset, including creating and modifying versions and assets.

File: app/routes/datasets/versions.py [hunk 3]
```diff
@@ -124,7 +125,10 @@ async def add_new_version(
     is_authorized: bool = Depends(is_admin),
     response: Response,
 ):
-    """Create or update a version for a given dataset."""
+    """Create a version for a given dataset uploading the geospatial/tabular asset.
```
Comment (danscales) [inline app/routes/datasets/versions.py:128]: maybe insert 'by' (or possibly 'after') between 'dataset' and 'uploading'.

**Review by danscales — APPROVED** [review]
Thanks for all the added comments, in addition to the permission comments!

**Review by dmannarino — APPROVED** [review]
Great, thanks! I would echo Dan's suggestions, on top of which I have none!

**Review by jterry64 — APPROVED** [review]
Looks good! Only small suggestion, "parent dataset's owner" is a bit of a mouthful, would just "dataset owner" be clear enough?

**Pull request merged**